app/controllers/questions_controller.rb

```diff
@@ -13,12 +16,17 @@ def index
     render json: questions
   end
 
-  def set_question
-    @question = Question.find(params[:id])
-    return render json: { message: 'Question not found' }, status: :not_found if @question.nil?
+  def show
+    question = Question.find_by(user_id: @user_id)
+    render json: question
   end
```

Comment:
Show action should show the question when any id is passed. It should not check whether the question belongs to the current user or not.